### DISCUSSION TITLE
Sling's relation with DBT

### DISCUSSION THREAD

**Discussion by ShahBinoy** [slingdata-io/sling-cli#104] [General]
Hi,
So Sling supports reading DBT configuration file, but the document does not mention any interaction between Sling and DBT. Is there any hidden/planned feature that will allow the interaction or are they to be treated as separate tools that the users will have to orchestrate.

Tool separation has its advantages to no qualms there but, Sling inherently parsing DBT profile was a surprise discovery and thus this question.

**Comment by flarco** [comment]
No plans. I use dbt alot, so it was convenient to have it read the profiles. But for now I think it's best if it's left as a separate process. 

Just curious, what kind of interaction between sling & dbt would you find useful?

**Reply by ShahBinoy** [reply]
Nothing as of right now, I was just looking to understand the reason why dbt's configuration is recognized

Thanks for the explanation
